sdks/python/apache_beam/runners/dataflow/internal/dependency_test.py

```diff
@@ -42,6 +42,18 @@
 @unittest.skipIf(HttpError is None, 'GCP dependencies are not installed')
 class SetupTest(unittest.TestCase):
 
+  def setUp(self):
+    self._temp_dir = None
+
+  def temp_dir(self):
```

Comment:
How about s/temp_dir/make_temp_dir/  since we are creating new temp directory every time we call the method?

Reply:
or create_temp_dir for consistency with create_temp_file.

Reply:
Thanks. Yes, that's a better name. Done.